### PR TITLE
WHO IS JOHN NECRA WHY DOES HE WEAR THE MASK

### PR DESCRIPTION
**Pull request by YuiY1997**
## About The Pull Request
Makes the new unique Necra cleric mask a mask

Moves the deathmask sprite to the appropriate folder
## Why It's Good For The Game
Allows the equipping of helmets while wearing it. It's a good unique item, but is basically outclassed by every helmet in the game.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

test

### CODE REVIEW

**Comment by YuiY1997** [comment]
I didn't know there were any that were mapped lmao uh oh

File: code/modules/clothing/rogueclothes/hats.dm [hunk 1]
```diff
@@ -260,17 +260,6 @@
...
-	name = "death shroud"
-	desc = "When inducted into the cult of Necra, the supplicant must make a talisman from the remains of a deceased loved one. Many favor a chin-guard made from a jawbone."
-	icon_state = "deathface"
-	flags_inv = HIDEEARS | HIDEHAIR | HIDEFACIALHAIR
-
-	armor = ARMOR_WEAK
-	prevent_crits = MINOR_CRITICALS
-
 //................ Death Shroud ............... //	- Necra headwear that conceals indentity
 /obj/item/clothing/head/roguetown/padded/deathshroud
```
Comment (BadAtThisGame302) [inline code/modules/clothing/rogueclothes/hats.dm:264]: I love how there's two deathshrouds, so fucking bleak.

**Comment by BadAtThisGame302** [comment]
Need to remove that one too and replace it aswell.

**Comment by YuiY1997** [comment]
> Need to remove that one too and replace it aswell.

image

Donezo

**Comment by YuiY1997** [comment]
image
goes hard with armor

**Comment by YuiY1997** [comment]
okay so the item was on a bunch of maps I didn't know about. Fixed.

**Comment by NPC1314** [comment]
on one hand it does make sense and it doesnt look bad, on the other it kinda breaks with the whole "all priests/acolytes supposed to wear their unique headgear to make them visually distinct" 
Need to give this one a thonk.

**Comment by YuiY1997** [comment]
> on one hand it does make sense and it doesnt look bad, on the other it kinda breaks with the whole "all priests/acolytes supposed to wear their unique headgear to make them visually distinct" Need to give this one a thonk.

Well a lot of the Necra Acolyte stuff was just moved to gravedigger anyway so that's worth taking into consideration.

**Pull request closed without merging**